pytest_mypy/item.py

```diff
@@ -180,12 +189,10 @@ def typecheck_in_same_process(self, execution_path: Path, mypy_cmd_options: List
                 # add current directory to path
                 sys.path.insert(0, str(execution_path))
 
-                with capturer.CaptureOutput(merged=False) as captured_std_streams:
+                with capture_std_streams() as (stdout, stderr):
```

Comment:
Just FYI, I believe I noticed `typecheck_in_same_process` isn't covered by the tests